lib/huginn_acumen_product_agent/concerns/inv_product_query_concern.rb

```diff
@@ -33,6 +33,15 @@ def process_inv_product_response(raw_data, digital_format_list)
             'Inv_Product.Next_Release' => 'releaseDate',
         })
 
+        # Nullify blank dates
+        if product['datePublished'] == '0000-00-00T00:00:00'
```

Comment:
Are these always in this format, but are missing a timezone instead of `:00Z`? You could substring it and just check the date part.

Reply:
Based on what I see coming from the API, this appears to be how Acumen returns data when a date field is blank.